### PR TITLE
Donut Chart Enhancements

### PR DESCRIPTION
**Pull request by dtaylor113**
- Added onClickFn
- Updated size to match latest PF ref. impl.
- Removed styles in angular-pf which are now in base patternfly
- Clearer ngdocs:

image

### CODE REVIEW

File: dist/angular-patternfly.js [hunk 1]
```diff
@@ -691,11 +691,12 @@ angular.module('patternfly.card').directive('pfCard', function () {
...
- * <li>.units         - unit label for values, ex: 'MHz','GB', etc..
- * <li>.thresholds    - warning and error percentage thresholds used to determine the Usage Percentage fill color (optional)
- * <li>.tooltipFn     - user defined function to customize the tool tip (optional)
- * <li>.centerLabelFn - user defined function to customize the center label (optional)
+ * <li>.chartId        - the unique id of the donut chart
+ * <li>.units          - unit label for values, ex: 'MHz','GB', etc..
+ * <li>.thresholds     - warning and error percentage thresholds used to determine the Usage Percentage fill color (optional)
+ * <li>.tooltipFn(d)   - user defined function to customize the tool tip (optional)
+ * <li>.centerLabelFn  - user defined function to customize the center label (optional)
+ * <li>.onClickFn(d,i) - user defined function to handle when donut arc is clicked upon.
```
Comment (jeff-phillips-18) [inline dist/angular-patternfly.js]: Need a unit test for this

Comment (dtaylor113) [inline dist/angular-patternfly.js]: Tried, same old issue, can't unit test the C3 charts ;-(

Comment (jeff-phillips-18) [inline dist/angular-patternfly.js]: Ah yes...

**Comment by jeff-phillips-18** [comment]
LGTM (after you fix the eslint issues caused by updated version of grunt-eslint)

**Comment by dtaylor113** [comment]
@erundle please review and merge if happy with it ;-)

**Pull request merged**